src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form class="form">
+      <div class="personalInfo">
```

Comment:
why do we need div here?

Reply:
it's just a convenient wrapper for grouping content that does not affect anything until it is styled